proxy/config.go

```diff
@@ -3,8 +3,18 @@ package proxy
 import (
 	"bufio"
 	"fmt"
+	"log"
 	"os"
 	"regexp"
+	"sync"
+	"time"
+)
+
+var (
+	configFile string
```

Comment:
I think we can move this to `Config` struct. What do you think?

Reply:
Yes, but then you would have to somehow add it to config. In LoadServices you do not have a reference to config and it is there that you add the config file. So you would have to either create a struct type that will have all the methods (LoadServices, GetService and AddService) or you pass the config in the parameters of the LoadServices function. 
I will have a look at it.